autopush/websocket.py

```diff
@@ -896,6 +903,12 @@ def error_notification_overload(self, fail):
         d = self.deferToLater(randrange(5, 60), self.process_notifications)
         d.addErrback(self.trap_cancel)
 
+    def error_message_overload(self, fail):
+        """errBack for handling excessive messages per UAID"""
+        fail.trap(MessageOverloadException)
+        self.ap_settings.router.drop_user(self.ps.uaid)
+        self.send_close()
```

Comment:
maybe we should make websocket tests' tearDown fail the test if a log.failure is called and wasn't marked as being expected

since websocket ends up eating exceptions and just closing the connection in some cases